src/highdicom/enum.py

```diff
@@ -29,9 +29,23 @@ class DimensionOrganizationTypeValues(Enum):
     TILED_SPARSE = 'TILED_SPARSE'
 
 
+class PatientSexValues(Enum):
+
+    """Enumerated values for Patient's Sex attribute."""
+
+    MALE = 'M'
+    FEMALE = 'F'
+    OTHER = 'O'
```

Comment:
For consistency with the other enums, I would suggesting using the abbreviations here, too (`M`, `F`, `O`).

Reply:
Hmm. I was going with the precedent of the enum immediately above using `THREE_DIMENSIONAL = '3D'`, but now I think about it I assume that was because beginning a variable with a number is disallowed, and not for clarity. Personally I think using clearer enums would lead to clearer code and fewer mistakes but I guess the ship has sailed now. 

Resolved in 3525c76